app/decorators/application_decorator.rb

```diff
@@ -12,4 +12,8 @@ def api_selector_api_link
   def self.collection_decorator_class
     PaginatingDecorator
   end
+
+  def updated_at
+    object.updated_at.to_s :long if object.updated_at?
```

Comment:
Where is all this called/used from? 🤔

Reply:
This parses `updated_at` to a human friendly format, depending on the user's time zone when calling `to_json`.